api/client.js

```diff
@@ -464,5 +471,31 @@ function uncapitalize(string) {
   return string.charAt(0).toLowerCase() + string.slice(1);
 }
 
+/**
```

Comment:
Can we reuse the one in transform.js and remove this one?

Reply:
Yes I had actually thought the same, however the callback return value when there is no return value in successful calls differed and the tests failed. 

After a discussion we decided to return any `undefined` or `null` return values in the api to mirror the same functionality in the client.